python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py

```diff
@@ -596,43 +629,69 @@ def resolve_pipelineRunTags(self, graphene_info: ResolveInfo):
     def resolve_runGroupOrError(self, graphene_info: ResolveInfo, runId):
         return get_run_group(graphene_info, runId)
 
-    def resolve_isPipelineConfigValid(self, graphene_info: ResolveInfo, pipeline, **kwargs):
+    def resolve_isPipelineConfigValid(
+        self,
+        graphene_info: ResolveInfo,
+        pipeline: GraphenePipelineSelector,
+        mode: str,
+        runConfigData: Optional[GrapheneRunConfigData] = None,
```

Comment:
[1] since GrapheneRunConfigData represents a "custom scalar", i don't think using that type here is right. Theoretically we could implement more on that class and align to clearer types that way. For now might be best to just do Any / Optional[object] / what we expect which is what `parse_run_config_input` is typed to.

https://docs.graphene-python.org/en/latest/types/scalars/#custom-scalars